### PR TITLE
Expert Progression

### PR DESCRIPTION
**Pull request by MuteTiefling**
# Misc

## Nomadic Tents
All tent crafting has been simplified so that far less micro crafting is required. 
- Tents now come with the floor layers maximized by default instead of needing to craft with a shovel.
- Crossbeams are also removed, allowing the highest tier to be crafted for much less. 
- Quests created to spotlight and explain tents.

Each type and tier follows the same pattern with wall sections and sticks surrounding the previous tier tent. 
image

## Void Safety
When players fall into the void in the Nomadic Tents dimension, Compact Machines dimension, or The End, they'll be warped back to their spawn safely without taking damage. This is largely to prevent issues retrieving graves in hard to reach places when a bug yeets you where you shouldn't be. 

The End is included as well since retrieving a grave thousands of blocks into the end in the middle of the void is just annoying. 

## Villagers
Removed disabled items from IE villagers
Added new fun trades in their places
Removed trades from AE2 villager (Point of Interest was already disabled and trades added to PNC Villager)

## Armor Set Buffs
Experimental.

- When wearing a full set of Pneumatic Armor, the player gets a Mana Regen 3. This largely just helps it stay relevant compared to Ars armor. 
- When wearing any part of the meka suit, the player gets Hymn of Order (disables casting) and Summoning Sickness (cannot summon with Ars). This is to counterbalance the fact that the armor makes you 100% immune to magic when powered.
- Ars Elemental armor sets, when wearing all pieces for a set, grant the player Spell Damage 3 to really ramp up their potential when wearing this end game armor.
- Beekeeper armor offers the Hidden buff, which makes bees less interested in you. 

MekaSuit power costs altered heavily. Each armor piece holds significantly less FE, requiring the player to keep it charged via Player Transmitters. Certain damage costs are high relative to the amount stored. Some being higher than can be stored without 4+ power upgrades, thereby requiring upgrades to really work well. Charge Rates are also reduced, creating an actual bottleneck to keeping the suit powered, allowing it to potentially be drained when under heavy assault. 

## Portals
Disables Nether/End portal formation. Initial travel must now happen via Ritual

### CODE REVIEW

**Comment by MuteTiefling** [comment]
# Magic

**Comment by MuteTiefling** [comment]
# Tech

**Comment by MuteTiefling** [comment]
# Progression

Tree of Life 2 is now summoned after helping the Queen
image

This enables travel to the Nether by way of ritual after obtaining a Nether Star and Celestial Gold
image

**Comment by MuteTiefling** [comment]
Going to leave this as is, since it has some nice updates for normal as well (change log updated) that would be good to get into the next release.

**Comment by MuteTiefling** [comment]
Required mods
- ftb-quests-forge-1902.4.16-build.235
- morejs-forge-1.19.2-0.1.1

File: kubejs/server_scripts/base/player_events/void_safety_net.js
Comment (NielsPilgaard) [inline kubejs/server_scripts/base/player_events/void_safety_net.js:1]: Is this file meant to be here? Seems like it's a duplicate of the one above, just with incorrect path/file naming.

**Review by NielsPilgaard — APPROVED** [review]

File: kubejs/server_scripts/base/player_events/void_safety_net.js
Comment (MuteTiefling) [reply kubejs/server_scripts/base/player_events/void_safety_net.js:1]: Nope. goofed up the move.

**Review by NielsPilgaard — APPROVED** [review]

**Pull request merged**